service.go

```diff
@@ -387,6 +387,19 @@ type Service interface {
 	Status() (Status, error)
 }
 
+// ConfigInfoer is an optional interface which allows for certain information to be obtained.
+type ConfigInfoer interface {
+	configpath() (string, error)
```

Comment:
Export this as `ConfigPath() (string, error)`

Reply:
As requested